courses/machine_learning/deepdive2/how_google_does_ml/solutions/analyze_with_bigquery_solution.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p> </p><ul><li> Pass the query through a sql formatter (e.g. <a href="http://www.dpriver.com/pp/sqlformat.htm" target="_blank">http://www.dpriver.com/pp/sqlformat.htm</a> generic)</li><li>Change the scenario to a ml one (or remove it)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>As will be pointed out in an earlier cell, the scenario is to become acquainted with BigQuery to prepare data for ML usage downstream.  The data and outcomes in this lab will not be used in any other lab.</p><p> </p><p>The lab will be very, very short if we remove these "SQL" activities.</p><p> </p><p>Query reformatted.</p>